solr/core/src/java/org/apache/solr/cli/AuthTool.java

```diff
@@ -75,8 +75,10 @@ public class AuthTool extends ToolBase {
           .longOpt("solr-include-file")
           .hasArg()
           .argName("FILE")
+          .required()
```

Comment:
Looking into our source code, this is actually required and always set via the CLI script, and if not set it would likely fail with a NPE I believe?

What I am wondering though is, should this be required? Does the file always exist and does it contain information we need? Or should we simply handle cases where it is not defined?

Reply:
At least right now, we have logic that if this file does NOT exist or is not writable, then the AuthTool quits with a warning.   So I think making it required is the right thing.   Having said that, longer term maybe something to think about?   Becasue it makes it implicitly assumed that the bin/solr auth command runs on your Solr server!